### PR TITLE
[Documenation] Turn off bundles fixes

### PR DESCRIPTION
**Pull request by lchrusciel**
| Q               | A
| --------------- | -----
| Branch?         | 1.0 
| Bug fix?        | yes
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | 
| License         | MIT

### CODE REVIEW

File: docs/cookbook/configuration/disabling-shop-admin-api.rst [hunk 1]
```diff
@@ -52,13 +52,16 @@ The part that has to be removed from this file is shown below:
 
 .. code-block:: yaml
 
+    parameters:
+        # sylius.security.shop_regex: "^/(?!admin|api/.*|api$)[^/]++"
```
Comment (pamil) [inline docs/cookbook/configuration/disabling-shop-admin-api.rst]: Is it a valid YAML syntax?

```yaml
parameters:
# nothing here
security:
# ...
```

Comment (Zales0123) [reply docs/cookbook/configuration/disabling-shop-admin-api.rst]: Probably something can go wrong with that :/

File: docs/cookbook/configuration/disabling-shop-admin-api.rst [hunk 2]
```diff
@@ -82,6 +87,18 @@ The part that has to be removed from this file is shown below:
     #                success_handler: sylius.handler.shop_user_logout
     #            anonymous: true
 
+    access_control:
```
Comment (Zales0123) [inline docs/cookbook/configuration/disabling-shop-admin-api.rst]: I think the same here, and for `security.firewalls` I think they all should have `~` at the end... @lchrusciel could you check that?

**Comment by lchrusciel** [comment]
I just commented the lines that should be removed. In each of the sections, there are a few more configurations, and I didn't want to copy and paste the whole security setup because it is too verbose for these examples. /cc @pamil @Zales0123

**Comment by pamil** [comment]
Thanks Łuki! 🌮

**Pull request merged**